frontend/src/forms/manage-restaurant-form/CuisineCheckBox.tsx

```diff
@@ -0,0 +1,33 @@
+import { Checkbox } from '@/components/ui/checkbox';
+import { FormControl, FormItem, FormLabel } from '@/components/ui/form';
+import { ControllerRenderProps, FieldValues } from 'react-hook-form';
+
+type Props = {
+  cuisine: string;
+  field: ControllerRenderProps<FieldValues, 'cuisines'>;
+};
+
+const CuisineCheckBox = ({ cuisine, field }: Props) => {
+  return (
+    <FormItem className="flex flex-row items-center space-x-1 space-y-0 mt-2">
+      <FormControl>
+        <Checkbox
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/29#issuecomment-2385686111>#1c596c</a><span> - 10/01/2024, 12:49 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The component lacks proper accessibility attributes, which may make it difficult for users with disabilities to interact with the form. A similar issue was also found in frontend/src/components/ui/checkbox.tsx (line 17).
</div>
<div id="fix">
<b>Fix:</b> Add appropriate ARIA attributes to the Checkbox component to improve accessibility. This includes adding an aria-label or aria-labelledby attribute to provide a descriptive label for screen readers.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -13,6 +13,8 @@
    <FormControl>
      <Checkbox
        className="bg-white"
 +      aria-label={'Select ${cuisine} cuisine'}
 +      id={'cuisine-${cuisine}'}
        checked={field.value.includes(cuisine)}
        onCheckedChange={(isChecked) => {
          if (isChecked) {
 @@ -25,7 +27,7 @@
        }}
      />
    </FormControl>
 -  <FormLabel className="text-sm font-normal">{cuisine}</FormLabel>
 +  <FormLabel htmlFor={'cuisine-${cuisine}'} className="text-sm font-normal">{cuisine}</FormLabel>
  </FormItem>
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged